### ISSUE TITLE
Confusion with Heliocentric Cartesian vs Heliocentric Radial angles

### ISSUE THREAD

**Issue by hayesla** [sunpy/sunpy#5240]
### Description
Maybe I'm missing something, but I'm looking to convert a HPC coordinate to the HCR frame. In the docs it says for HCR to use HCC with "Use a cylindrical representation, but with a 90-degree offset in psi", as this is due to the `psi` value in HCC measured from the west limb rather than solar north for HCR. 

However, when looking at the HCC coordinates it seems to map `psi` from +180, -180 rather than from 0-360, hence the offset of 90 for psi can get confusing as HCR goes from 0-360 according to the Thomson paper. 

I know this can be easily worked out what the HCR `psi` should be, but it still a little confusing and I spent a bit of time looking through the source code to figure out what was going on (and drawing many circles haha). 

Is there a way to force the HCC `psi` values to go from 0-360? If not I think that this should be clearer in the docs.

### Example
For example, I think the issue lies that even if you define a coordinate in HCC with a +180 degrees it converts e.g.
```python
>>> hcc = SkyCoord(CylindricalRepresentation(R_sun, 190*u.deg, 0*u.km),
                   frame=frames.Heliocentric(observer="earth", obstime=date))

>>> hcc.cylindrical.phi
<CylindricalRepresentation (rho, phi, z) in (m, deg, km)
    (6.957e+08, -170., 0.)>
```
So its changing the 190 degrees to -170. Naively one may assume that the HCR phi would just be a 90 degree offset from -170, whereas its actually (-170 + 360) with offset of 90. I know this is super simple as its just a circle but it took me an embarrassingly long time to figure out what was going on when I was converting HPC coords to HCC to get the HCR psi value 🤦‍♀️ .
 
### In Summary
What I'm looking for is to convert a list to HPC coords to HCR to get the `psi` values,  and my life would be easier if this `psi` value could be in 0-360 range rather than -180, 180 and this made me confused. Maybe in astropy you can easily do this with an angle? I played around the `_wrap_angle` keyword with no avail.

**Comment by hayesla** [comment]
oh I just realised you can just use the `wrap_at(360*u.deg)` 

```python
>>> hcc.cylindrical.phi.wrap_at(360*u.deg)
```
and it will give you what I'm looking for. 

However just a general note - can we make this wrapping default for the HCC frame?

**Comment by ayshih** [comment]
See #5242 

FYI, the `_wrap_angle` variable affects only spherical coordinates.
